curriculum/challenges/english/25-front-end-development/quiz-javascript-fundamentals/66edcd875b0d91de1fbbb492.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
+Destructure an object into parameters
 
 #### --answer--
 
-Placeholder answer
+Capture all arguments into one array
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is the correct syntax for the rest parameter in JavaScript?
 
 #### --distractors--
 
-Placeholder distractor 1
+`function myFunction(restParams)`
 
 ---
 
-Placeholder distractor 2
+`function myFunction(restParams...)`
 
 ---
 
-Placeholder distractor 3
+`function myFunction([...restParams])`
 
 #### --answer--
 
-Placeholder answer
+`function myFunction(...restParams)`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does destructuring allows you to do in JavaScript?
 
 #### --distractors--
 
-Placeholder distractor 1
+Build complex objects
 
 ---
 
-Placeholder distractor 2
+Serialize objects
 
 ---
 
-Placeholder distractor 3
+Merge arrays or objects
 
 #### --answer--
 
-Placeholder answer
+Extract properties from objects and elements from arrays
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a valid function name in JavaScript?
 
 #### --distractors--
 
-Placeholder distractor 1
+`123function`
 
 ---
 
-Placeholder distractor 2
+`function-name`
 
 ---
 
-Placeholder distractor 3
+`function`
 
 #### --answer--
 
-Placeholder answer
+`functionName`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of these is NOT a good practice for naming variables and functions in JavaScript?
 
 #### --distractors--
 
-Placeholder distractor 1
+Use camelCase notation
 
 ---
 
-Placeholder distractor 2
+Start variable names with a letter, $ or _
 
 ---
 
-Placeholder distractor 3
+Using meaningful names
 
 #### --answer--
 
-Placeholder answer
+Using reserved keywords as variable names
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the primary purpose of a linter in JavaScript development?
 
 #### --distractors--
 
-Placeholder distractor 1
+To format code automatically
 
 ---
 
-Placeholder distractor 2
+To optimize code for performance
 
 ---
 
-Placeholder distractor 3
+To create documentation for code
 
 #### --answer--
 
-Placeholder answer
+To detect potential errors and style issues in code
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is NOT a linter for JavaScript?
 
 #### --distractors--
 
-Placeholder distractor 1
+ESLint
 
 ---
 
-Placeholder distractor 2
+standardJS
 
 ---
 
-Placeholder distractor 3
+quick-lint-js
 
 #### --answer--
 
-Placeholder answer
+Babel
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of these describes accessibility?
 
 #### --distractors--
 
-Placeholder distractor 1
+Making sure code is accessible to other developers
 
 ---
 
-Placeholder distractor 2
+Writing secure code
 
 ---
 
-Placeholder distractor 3
+Controlling who can access certain data
 
 #### --answer--
 
-Placeholder answer
+Making web interfaces usable by all, including people with disabilities
```

Comment:
I think this question will be better if it dealt with Accessibility in a Javascript context rather than a general one. 

https://developer.mozilla.org/en-US/curriculum/core/accessibility/#7.3_accessible_javascript